cmd/influxd/launcher/query_test.go

```diff
@@ -771,87 +881,92 @@ m0,k=k0 f=5i 12000000000
 m0,k=k0 f=8i 13000000000
 m0,k=k0 f=9i 14000000000
 m0,k=k0 f=5i 15000000000
-`)
-
-	getReadRequestCount := func() uint64 {
-		const metricName = "query_influxdb_source_read_request_duration_seconds"
-		mf := l.Metrics(t)[metricName]
-		if mf != nil {
-			for _, m := range mf.Metric {
-				for _, label := range m.Label {
-					if label.GetName() == "op" && label.GetValue() == "readWindowAggregate" {
-						return m.Histogram.GetSampleCount()
-					}
-				}
-			}
-		}
-		return 0
-	}
-
-	for _, tt := range []struct {
-		name string
-		q    string
-		res  string
-	}{
-		{
-			name: "count",
-			q: `
-from(bucket: v.bucket)
-	|> range(start: 1970-01-01T00:00:00Z, stop: 1970-01-01T00:00:15Z)
-	|> aggregateWindow(every: 5s, fn: count)
-	|> drop(columns: ["_start", "_stop"])
 `,
-			res: `
-#datatype,string,long,dateTime:RFC3339,long,string,string,string
-#group,false,false,false,false,true,true,true
-#default,_result,,,,,,
-,result,table,_time,_value,_field,_measurement,k
-,,0,1970-01-01T00:00:05Z,5,f,m0,k0
-,,0,1970-01-01T00:00:10Z,5,f,m0,k0
-,,0,1970-01-01T00:00:15Z,5,f,m0,k0
+			want: `,result,table,_time,_value,_field,_measurement,k
+,_result,0,1970-01-01T00:00:05Z,5,f,m0,k0
+,_result,0,1970-01-01T00:00:10Z,5,f,m0,k0
+,_result,0,1970-01-01T00:00:15Z,5,f,m0,k0
 `,
+			op: "readWindowAggregate",
 		},
 		{
 			name: "bare count",
-			q: `
-from(bucket: v.bucket)
+			query: `
+from(bucket: "bucket")
 	|> range(start: 1970-01-01T00:00:00Z, stop: 1970-01-01T00:00:15Z)
 	|> count()
 	|> drop(columns: ["_start", "_stop"])
 `,
-			res: `
-#group,false,false,false,true,true,true
-#datatype,string,long,long,string,string,string
-#default,_result,,,,,
-,result,table,_value,_field,_measurement,k
-,,0,15,f,m0,k0
+			data: `
+m0,k=k0 f=0i 0
+m0,k=k0 f=1i 1000000000
+m0,k=k0 f=2i 2000000000
+m0,k=k0 f=3i 3000000000
+m0,k=k0 f=4i 4000000000
+m0,k=k0 f=5i 5000000000
+m0,k=k0 f=6i 6000000000
+m0,k=k0 f=5i 7000000000
+m0,k=k0 f=0i 8000000000
+m0,k=k0 f=6i 9000000000
+m0,k=k0 f=6i 10000000000
+m0,k=k0 f=7i 11000000000
+m0,k=k0 f=5i 12000000000
+m0,k=k0 f=8i 13000000000
+m0,k=k0 f=9i 14000000000
+m0,k=k0 f=5i 15000000000
+`,
+			want: `,result,table,_value,_field,_measurement,k
+,_result,0,15,f,m0,k0
 `,
+			op: "readWindowAggregate",
 		},
-	} {
-		t.Run(tt.name, func(t *testing.T) {
-			wantCount := getReadRequestCount() + 1
-
-			prelude := fmt.Sprintf("v = {bucket: \"%s\", timeRangeStart: 1970-01-01T00:00:00Z, timeRangeStop: 1970-01-01T00:00:15Z}", l.Bucket.Name)
-			queryStr := prelude + "\n" + tt.q
-			res := l.MustExecuteQuery(queryStr)
-			defer res.Done()
-			got := flux.NewSliceResultIterator(res.Results)
-			defer got.Release()
-
-			dec := csv.NewMultiResultDecoder(csv.ResultDecoderConfig{})
-			want, err := dec.Decode(ioutil.NopCloser(strings.NewReader(tt.res)))
-			if err != nil {
-				t.Fatal(err)
-			}
-			defer want.Release()
+	}
 
-			if err := executetest.EqualResultIterators(want, got); err != nil {
```

Comment:
This test used to compare the iterators instead of just comparing the raw CSV, which is more fragile. My understanding was that in OSS we didn't have problems with non-determinism comparing with result iterators. Is it possible to change this back?